src/primitives/block.h

```diff
@@ -92,7 +92,7 @@ class CBlock : public CBlockHeader
     ADD_SERIALIZE_METHODS;
 
     template <typename Stream, typename Operation>
-    inline void SerializationOp(Stream& s, Operation ser_action, int nType, int _nVersion) {
+    inline void SerializationOp(Stream& s, Operation ser_action, int nType, int nVersion) {
```

Comment:
Hmm, this particular change is causing my clang on OS X build to fail `-rescan` with:

```
2016-09-26 19:53:08 ERROR: ReadBlockFromDisk: Deserialize or I/O error - ReadCompactSize(): size too large: unspecified iostream_category error at CBlockDiskPos(nFile=49, nPos=4208853)
```

So reverting it.

But do not ask me, why it is the cause... I do not know yet.